api/src/controllers/webfinger.js

```diff
@@ -133,7 +133,8 @@ function WebfingerControllerFactory(log, config, ledger) {
       this.body = {
         'subject': config.data.get('client_host'),
         'properties': {
-          'https://interledger.org/rel/publicKey': config.data.getIn(['connector', 'public_key'])
+          'https://interledger.org/rel/publicKey': config.data.getIn(['connector', 'public_key']),
+          'https://interledger.org/rel/protocolVersion': 'Compatible: ilp-kit v' + config.data.getIn(['ilpKitVersion'])
```

Comment:
I think the solution would be to use semantic-release. Then every release is automatically tagged with a version number.

Of course compatibility should only depend on the major version - in theory.